### DISCUSSION TITLE
part-ii/2015-91/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#282] [Question comments]
Part II 2015 Number Theory: Paper 3, Section I, H 

https://questions.tripos.org/part-ii/2015-91/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

A positive definite BQF $(a,b,c)$ is reduced if either $0 \leq b \leq a = c$ or $-a < b \leq a < c$.

To study the forms $f$ and $g$, we find reduced forms they are equivalent to. We use the following operations, which send BQFs to equivalent forms:

$$
S: (a,b,c) \to (c,-b,a) \\
T_{\pm}: (a,b,c) \to (a, b \pm 2a, a \pm b + c) \\
$$

For $f$, we get 

$$(3,8,9) \to (3,2,4)$$

and for $g$, we get

$$(15,34,20) \to (15,4,1) \to (1,-4,15) \to (1,-2,12) \to (1,0,11)$$

Now the smallest positive integers properly represented by a positive definite BQF $(a,b,c)$ are $a,c,a-|b|+c$, and equivalent forms represent the same integers, so for $f$, we get $3,4,5$, and for $g$, we get $1,11,12$.

Finally, we compute all of the reduced positive definite BQFs with discriminant $d = -44$. We use the following result. 

Lemma: If $(a,b,c)$ is a reduced positive definite BQF with discriminant $d$, then $|b| \leq a \leq \sqrt{\frac{|d|}{3}}$ and $b \equiv d \pmod{2}$.

For $d = -44$, we have that $\sqrt{\frac{|d|}{3}} < \sqrt{\frac{48}{3}} = \sqrt{16} = 4$. Therefore $|b| < 4$, and since $|b| \equiv 0 \pmod{2}$ by the last part of the lemma, either $|b| = 2$ or $|b| = 0$.

If $|b| = 2$, the inequality gives us that either $a = 2$ or $a = 3$. We know that $d = b^2 - 4ac$, so $-44 = 4 - 4ac$, so $ac = 12$. If $a = 2$, this gives $c = 6$, and if $a = 3$, this gives $c = 4$. We get the forms 

$$(2,2,6), (2,-2,6), (3,2,4), (3,-2,4)$$

But the second is not reduced, since it has $b = -a$.

If $|b| = 0$ (i.e. $b = 0$), then $a = 0,1,2,3$. We have that $-44 = -4ac$, so $ac = 11$, and of the previous possibilities, only $a = 1$ gives an integer value for $c$. This gives the form $(1,0,11)$.

So we are left with $(2,2,6),(3,-2,4),(3,2,4),(1,0,11)$. The first only represents even integers, and the second and third represent the same integers (just by replacing $x$ with $-x$). Therefore all odd integers represented by any form of discriminant $-44$ are represented by the third or fourth forms, so they are reprented by $f$ or $g$ (since these are the reduced forms $f$ and $g$ are equivalent to).

</details>
